### DISCUSSION TITLE
Resetting?

### DISCUSSION THREAD

**Discussion by macxwdc** [TodePond/CellPond#205] [Help] [answered]
How do I clear the board without losing the rules?

**Comment by TodePond** [comment]
Sorry it's not currently possible, but I'll be working on CellPond again after my next video, so hopefully I can add that in the future: https://github.com/TodePond/CellPond/issues/11

**Comment by i-rember** [comment, marked as answer]
Click anywhere outside the grid, then click again after 1.5 seconds.
